torch/_dynamo/variables/nn_module.py

```diff
@@ -305,6 +305,7 @@ def call_method(
         if name == "__call__":
             # TODO(whc)  do we really need this special case?
             return self.call_function(tx, args, kwargs)
+
         elif name == "forward":
```

Comment:
I think this is the right way:
1, update L309 to ```elif name in ["forward", "_call_impl"]:```
2, update L320 to:
```
if name == "forward":
    fn = module.forward.__func__
else:
    assert name == "_call_impl"
    fn = module._call_impl.__func__
```
3, Apply the same logic to ```ScriptModules```.
Then it would work well.

Reply:
We can still navigate the ```_call_impl``` call to the ```NNModuleVariable.call_method```, but we should inline there.